plonk/src/proof_system/structs.rs

```diff
@@ -100,7 +100,7 @@ where
                 .chunks_exact(2)
                 .map(|chunk| {
                     if chunk.len() == 2 {
-                        Commitment(Affine::new(chunk[0], chunk[1], false))
```

Comment:
I wonder if we should use this constructor or `Affine::new_unchecked` ( cuz the former include `is_on_curve` and `is_in_correct_subgroup` by default)